src/ucs/sys/sys.c

```diff
@@ -339,7 +340,7 @@ size_t ucs_get_page_size()
     static size_t page_size = 0;
 
     if (page_size == 0) {
-        page_size = ucs_sysconf(_SC_PAGESIZE);
+        page_size = ucs_max(ucs_sysconf(_SC_PAGESIZE), 4096);
```

Comment:
this is not a good way to treat -1. need to check for -1 explicitly. i see that ucs_get_max_iov() did that before your change, but this is not a good approach now.

Reply:
ucs_get_max_iov didn't do any check as far as I can tell. I don't see it anywhere in the code.

Reply:
Are you okay with 4k as a default ?

Reply:
yes